graphql/pupil/fields.ts

```diff
@@ -100,7 +100,7 @@ export class ExtendFieldsPupilResolver {
     @LimitEstimated(100)
     async logs(@Root() pupil: Required<Pupil>) {
         return await prisma.log.findMany({
-            where: { user: pupil.wix_id },
+            where: { userID: userForPupil(pupil).userID },
```

Comment:
Don't we have the same field resolver for students? Also while at it, we could maybe move this field resolver to the user entity

Reply:
We don't have it for students, but I'll move it to the user entity 👍